src/components/ListHeader/ListHeader.js

```diff
@@ -1,11 +1,12 @@
+/* eslint-disable  */
```

Comment:
I think we just added this whilst we were messing with some stuff, it can probably be removed now